### DISCUSSION TITLE
don't work too much :D

### DISCUSSION THREAD

**Discussion by acrylic-style** [oamaok/ezpp#106] [General]
overworking is bad for your body and mental
![image](https://user-images.githubusercontent.com/19150229/112655738-0f6c6c80-8e94-11eb-9e33-af8f3318229b.png)

P.S. I'm gonna sleep from now because it's 0:35 am for me, lol

**Comment by oamaok** [comment]
I have my day job as a software consultant to wear me out for the day, can't code too much :D
